### PR TITLE
Add QR code generation

### PR DESCRIPTION
**Pull request by mfucci**
This was much harder to write than I thought!

### CODE REVIEW

**Comment by Apollon77** [comment]
Wow, will review tomorrow: BTW The chip tool have that also as a link to a Webpage where the "qr code" is in the URL as textual representation ... We can maybe also add such a "URL" to the output later

**Comment by mfucci** [comment]
Done!
Now, when node-matter starts:

1. it prints the QR code
2. Gives the URL to the online version
3. Displays the manual pairing code

File: src/codec/PairingCode.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    /** Device supports BLE for discovery when not commissioned. */
+    ble: BitFlag(1),
+
+    /** Device is already on the IP network. */
+    onIpNetwork: BitFlag(2),
+});
+
+/** See {@link MatterCoreSpecificationV1_0} § 5.1.3.1 Table 35 */
+const QrCodeDataSchema = ByteArrayBitmapSchema({
```
Comment (Apollon77) [inline src/codec/PairingCode.ts:37]: Which mechanism adds the 4 bytes at the end of the struct? meaning the "Padding" from specs

**Review by Apollon77 — APPROVED** [review]
awesome

**Comment by lboue** [comment]
Can you merge with the master branch if it works fine?

File: src/codec/PairingCode.ts (hunk 1 above)
Comment (mfucci) [reply src/codec/PairingCode.ts:37]: The ByteArrayBitmapSchema sets to 0 all bits not defined.
Yes, I know: probably we should add a mechanism to explicitly declared reserve bits and their values...

**Pull request merged**